metafacture-io/src/main/java/org/metafacture/io/HttpOpener.java

```diff
@@ -98,7 +97,7 @@ private boolean getInputAsBody() {
      */
     public HttpOpener() {
         setAccept(ACCEPT_DEFAULT);
-        setContentType(CONTENT_TYPE_DEFAULT);
+        setContentType(ACCEPT_DEFAULT);
```

Comment:
This looks confusing. Why would the content-type default be tied to the accept default?

Reply:
added CONTENT_TYPE_DEFAULT on its own